hdr.cpp

```diff
@@ -65,10 +65,10 @@ void ExposureSeries::merge() {
 		cout << "Only one exposure was specified -- not doing HDR merging." << endl;
 
 		#pragma omp parallel for
-		for (size_t y=0; y<height; ++y) {
+		for (int y=0; y<height; ++y) {
```

Comment:
If you change this from size_t to int, then please also add a (size_t) cast at the places where the variable is used.

For instance ``y*width`` can overflow for truly big images (happened to me!),  write ``(size_t) y * width``